### PR TITLE
42 work flow configuration in one location

### PR DESCRIPTION
**Pull request by MarkTNO**

### CODE REVIEW

File: src/omotes_orchestrator/main.py [hunk 1]
```diff
@@ -519,33 +525,8 @@ def main() -> None:
...
-                workflow_type_name="simulator",
-                workflow_type_description_name="High fidelity simulator",
-            ),
-            WorkflowType(
-                workflow_type_name="test_worker",
-                workflow_type_description_name="Used for testing purposes. Should not be used in "
-                "production environments.",
-            ),
-        ]
+    workflow_type_manager = WorkflowTypeManager.from_json_config_file(
```
Comment (cwang39403) [inline src/omotes_orchestrator/main.py:528]: Not entirely clear to me why the `workflow_type_manager` is instantiated from a JSON config file here. I thought the workflow config would be centrally defined inside the `computation-engine`?

Comment (MarkTNO) [reply src/omotes_orchestrator/main.py:536]: The orchestrator reads this `../config/workflow_config.json` file which is volume mounted by the computation-engine: https://github.com/Project-OMOTES/computation-engine/blob/cb92353a12a77216945d0fccbe43e2d1ea59eba9/docker-compose.yml#L94

Comment (cwang39403) [reply src/omotes_orchestrator/main.py:536]: Ah, okay. Does it make sense to add a short description about the purpose of `config/workflow_config.json` to README under _Directory structure_?

Comment (MarkTNO) [reply src/omotes_orchestrator/main.py:536]: Already merged this PR but it is added here: 9af8641cd23e33e2cc568b44ecde74671d3509f0

**Pull request merged**